doc/Installation/Install-LibreNMS.md

```diff
@@ -261,7 +261,7 @@ Feel free to tune the performance settings in librenms.conf to meet your needs.
           try_files $uri $uri/ /index.php?$query_string;
          }
          location ~ [^/]\.php(/|$) {
-          fastcgi_pass unix:/run/php-fpm-librenms.sock;
+          fastcgi_pass unix:/run/php/php-fpm-librenms.sock;
```

Comment:
Docs above say;

Change `listen` to a unique name:
```
listen = /run/php-fpm-librenms.sock
```

I don't see what's wrong? Same path